src/sources/docker_logs.rs

```diff
@@ -790,6 +791,7 @@ impl ContainerLogInfo {
         let (stream, mut bytes_message) = match log_output {
             LogOutput::StdErr { message } => (STDERR.clone(), message),
             LogOutput::StdOut { message } => (STDOUT.clone(), message),
+            LogOutput::Console { message } => (CONSOLE.clone(), message),
             _ => return None,
```

Comment:
I feel like we could just make this `LogOutput::Stdin` for completeness given that all of the other enum variants are listed.

Reply:
Hello,

Thank you for your review :bow:
I added a dedicated test for a log sent by a container started with a tty.

For the LogOutput::Stdin I was also wondering if it wouldn't be better an explicit case and other match could lead to emitting a debug log because it would be easier to debug if bollard change  LogOutput. But I'm not sure if I am not overthinking it  :shrug: 
Maybe just adding LogOutput::Stdin => return None, would be sufficient ?

Reply:
Thanks for adding the test and fixing the `\r`!

It's actually not clear to me under what conditions the the log output would be `LogOutput::StdIn`. I think it'd be reasonable to just map it to `None` though which is the current state of things.

Reply:
I added the StdIn, now we will know if a pattern is not covered at compilation time :+1: